README.md

```diff
@@ -32,6 +32,7 @@ Copyright &copy; 2021 polarlight Team
 
 - [Chris Ohk](https://github.com/utilForever)
 - [James Chee](https://github.com/litcoderr)
+- [Jisang Han](https://github.com/onground-korea)
```

Comment:
Please remove it. I'll add new members by another PR.

Reply:
done